package.js

```diff
@@ -1,241 +1,174 @@
 Package.describe({
-  name: 'aldeed:autoform',
-  summary: 'Easily create forms with automatic insert and update, and automatic reactive validation.',
-  git: 'https://github.com/aldeed/meteor-autoform.git',
-  version: '6.3.0'
+  name: "aldeed:autoform",
+  summary:
+    "Easily create forms with automatic insert and update, and automatic reactive validation.",
+  git: "https://github.com/aldeed/meteor-autoform.git",
+  version: "6.3.0"
 });
 
 Package.onUse(function(api) {
-  api.versionsFrom('METEOR@1.3');
+  api.versionsFrom("METEOR@1.3");
 
   // Dependencies
-  api.use([
-    'livedata',
-    'underscore',
-    'deps',
-    'templating',
-    'ui',
-    'blaze',
-    'ejson',
-    'reactive-var',
-    'reactive-dict',
-    'random',
-    'jquery',
-    'ecmascript',
-    'momentjs:moment@2.10.6',
-  ], 'client');
+  api.use(
+    [
+      "livedata",
+      "deps",
+      "templating",
+      "ui",
+      "blaze",
+      "ejson",
+      "reactive-var",
+      "reactive-dict",
+      "random",
+      "ecmascript",
+      "momentjs:moment@2.10.6"
```

Comment:
@pouya-eghbali I checked with bundle visualizer and found 60kb due to the momentjs package being a hard dependency. Is there a reason against putting it into the weak dependencies?